_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title:  "Apprenticeship at 8th Light"
+date: 2017-10-05 14:23:00 +0200
+categories: how-we-learn 
+author_name: Wolfram Kriesing
+author_url : /author/wolframkriesing
+author_avatar: wolframkriesing
+show_avatar : true
+read_time : 20
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-05-8thlight-2/structured-design.jpg
+---
+
+Officially it was 2006 that 8th Light says they had started the [apprenticeship program]. From reading a bit around and searching for related articles online, it seems that there had been ocurrences of it before. Jake Scruggs [writes][jake-writes]  in 2009 about his apprenticeship that he had started 5 years before in 2004 at Object Mentor, kind of the predecessor of 8th Light. But that’s lot to cover, something for another history post :).
```

Comment:
I tend to get confused with times in English myself and one thing i see everyone running into is the wrong use of the *past perfect*: 

> The past perfect refers to a time earlier than before now. It is used to make it clear that one event happened before another in the past. It does not matter which event is mentioned first - the tense makes it clear which one happened first.

Of course I wouldn't be able to describe it this succinctly myself - the above is from http://www.ef.com/english-resources/english-grammar/past-perfect-tense/

Since in the phrase above there are no two events  that are put in relation to each other you should just use past tense: "Officially, the 8thlight apprenticeship program began as early as 2016".